common/app/views/fragments/analytics/google.scala.html

```diff
@@ -70,6 +75,7 @@
     @***************************************************************************************
     * Code that is customised by us                                                        *
     ***************************************************************************************@
+    window.guardian.config.page.gaIsInitalised = true;
```

Comment:
I guess we should be using the getters and setters from our config class here too, but making it accessible from this play template might be tricky?

Reply:
Yes, that won't be possible as-is. It's a nicety we'll have to live without.